go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -933,3 +957,20 @@ func calculateStaticMAC(podDesc string, mac string) (net.HardwareAddr, error) {
 	}
 	return podMac, nil
 }
+
+// getSwitchNames at some kubevirt scenarios the switch owner the IP is
+// different from the one running the pod
+func (bnc *BaseNetworkController) getSwitchNames(pod *corev1.Pod) (SwitchNames, error) {
+	switchNames := SwitchNames{}
+	var err error
+	switchNames.Current, err = bnc.getExpectedSwitchName(pod)
+	if err != nil {
+		return switchNames, fmt.Errorf("failed to get current switch name for pod %s/%s: %v", pod.Namespace, pod.Name, err)
+	}
+	switchNames.Original = switchNames.Current
+	originalSwitchName, ok := pod.Labels[kubevirt.OriginalSwitchNameLabel]
+	if kubevirt.PodIsLiveMigratable(pod) && ok {
```

Comment:
nit: I would name this `IsPodLiveMigratable` instead.

Reply:
You should read it including the package name "Kubevirt pod is live migratable" you are proposing "kubevir t is pod live migratable"

Reply:
Nah going with your proposal